doc/visualizations.rst

```diff
@@ -8,37 +8,81 @@ Scikit-learn defines a simple API for creating visualizations for machine
 learning. The key feature of this API is to allow for quick plotting and
 visual adjustments without recalculation. We provide `Display` classes that
 expose two methods for creating plots: `from_estimator` and
-`from_predictions`. The `from_estimator` method will take a fitted estimator
-and some data (`X` and `y`) and create a `Display` object. Sometimes, we would
-like to only compute the predictions once and one should use `from_predictions`
-instead. In the following example, we plot a ROC curve for a fitted support
-vector machine:
+`from_predictions`.
+
+The `from_estimator` method generates a `Display` object from a fitted estimator and
+input data (`X`, `y`).
+The `from_predictions` method creates a `Display` object from true and predicted
+values (`y_test`, `y_pred`). Using `from_predictions`
+avoids to recompute the predictions, but does not automatically resolve some
+ambiguities. The reason being that the user needs to know which column corresponds
```

Comment:
I think that we should explain that the story about the `pos_label` is only for binary classification.